detect.py

```diff
@@ -144,8 +145,10 @@ def run(
                 p, im0, frame = path, im0s.copy(), getattr(dataset, 'frame', 0)
 
             p = Path(p)  # to Path
-            save_path = str(save_dir / p.name)  # im.jpg
-            txt_path = str(save_dir / 'labels' / p.stem) + ('' if dataset.mode == 'image' else f'_{frame}')  # im.txt
+            relative_path_in_azure_mounted_folder = Path("/".join(p.parts[p.parts.index("wd")+2:]))
```

Comment:
can you add a comment to understand what this is pointing to? it's quite complicated

Reply:
Added.